webapp/src/shared/components/Row/ScoreRow.tsx

```diff
@@ -0,0 +1,103 @@
+import './Row.css';
+import { TextColor, TextVariant, TextWeight } from '../Text/Text';
+import { default as Text } from '../Text/Text';
+import { MediumAvatar } from '../Avatar/Avatar';
+import { Link } from 'react-router-dom';
+import { ScoreProps } from '../Score/Score';
+
+export type ScoreRowProps = {
+  scoreProps: ScoreProps;
+  url?: string;
+  gameMode?: string;
+  gameDuration?: number;
+  date?: Date;
+};
+
+export default function ScoreRow({
+  scoreProps,
+  url,
+  gameMode,
+  gameDuration,
+  date,
+}: ScoreRowProps) {
+  const cursorStyle = {
+    cursor: 'pointer',
+  };
+  const RowChildren: JSX.Element = (
+    <>
+      {scoreProps.playerOneAvatar && (
+        <div className="row-avatar-left">
+          <MediumAvatar {...scoreProps.playerOneAvatar} />
+        </div>
+      )}
+      <div className="scorerow_text_wrapper">
+        <div className="scorerow_text_upper_wrapper">
+          <Text
+            variant={TextVariant.HEADING}
+            color={TextColor.LIGHT}
+            weight={TextWeight.BOLD}
+          >
+            {scoreProps.playerOneUserName}
+          </Text>
+          <Text
+            variant={TextVariant.HEADING}
+            color={TextColor.LIGHT}
+            weight={TextWeight.BOLD}
+          >
+            {scoreProps.playerOneScore && scoreProps.playerTwoScore
+              ? `${scoreProps.playerOneScore?.toString()} - ${scoreProps.playerTwoScore?.toString()} `
+              : ' - '}
+          </Text>
+          <Text
+            variant={TextVariant.HEADING}
+            color={TextColor.LIGHT}
+            weight={TextWeight.BOLD}
+          >
+            {scoreProps.playerTwoUserName}
+          </Text>
+        </div>
+        <div className="scorerow_text_lower_wrapper">
+          {gameMode && (
+            <Text
+              variant={TextVariant.PARAGRAPH}
+              color={TextColor.LIGHT}
+              weight={TextWeight.REGULAR}
+            >
+              {`Mode: ${gameMode} `}
+            </Text>
+          )}
+          {gameDuration && (
+            <Text
+              variant={TextVariant.PARAGRAPH}
+              color={TextColor.LIGHT}
+              weight={TextWeight.REGULAR}
+            >
+              {`Duration: ${gameDuration}sec. `}
+            </Text>
+          )}
+          {date && (
+            <Text
+              variant={TextVariant.PARAGRAPH}
+              color={TextColor.LIGHT}
+              weight={TextWeight.REGULAR}
+            >
+              {`Date: ${date.toLocaleString('en-GB')}`}
+            </Text>
+          )}
+        </div>
+      </div>
+      {scoreProps.playerOneAvatar && (
+        <div className="row-avatar-right">
+          <MediumAvatar {...scoreProps.playerOneAvatar} />
```

Comment:
I think this line should be
```
          <MediumAvatar {...scoreProps.playerOneAvatar} />
```

I keep seeing the same avatar on the left and on the right.

Reply:
Sorry, done